dashboard/app/coverage_batch.go

```diff
@@ -0,0 +1,147 @@
+// Copyright 2017 syzkaller project authors. All rights reserved.
+// Use of this source code is governed by Apache 2 LICENSE that can be found in the LICENSE file.
+
+package main
+
+import (
+	"context"
+	"fmt"
+	"net/http"
+	"strconv"
+	"time"
+
+	batch "cloud.google.com/go/batch/apiv1"
+	"cloud.google.com/go/batch/apiv1/batchpb"
+	_ "cloud.google.com/go/bigquery"
+	"cloud.google.com/go/civil"
+	"github.com/google/uuid"
+	"google.golang.org/api/iterator"
+	"google.golang.org/appengine/v2"
+	"google.golang.org/appengine/v2/log"
+	"google.golang.org/protobuf/types/known/durationpb"
+)
+
+func initCoverageBatches() {
+	http.HandleFunc("/cron/batch_coverage", handleBatchCoverage)
+	// TODO: delme
+	http.HandleFunc("/test_batch_coverage", handleBatchCoverage)
```

Comment:
You can invoke cron urls in a browser, so  you can just use the /cron/batch_coverage for manual testing.

Reply:
done